src/Sylius/Component/Core/Formatter/StringInflector.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Sylius\Component\Core\Formatter;
 
+use Behat\Transliterator\Transliterator;
```

Comment:
I'm not sure if it is ok to use service from test dependency (Behat) in the Core component, wdyt @lchrusciel ?

Reply:
So in this case should we use same Transliterator in src/Sylius/Component/Product/Generator/SlugGenerator.php class? 🤔

Reply:
We already have this library in required in composer, so it is ok.
![image](https://user-images.githubusercontent.com/6213903/96549754-348afc80-12b0-11eb-944b-1453f471abce.png)